compose/install.md

```diff
@@ -31,27 +32,23 @@ Linux systems.
 ## Install Compose
 
 Follow the instructions below to install Compose on Mac, Windows, Windows Server
-2016, or Linux systems, or find out about alternatives like using the `pip`
-Python package manager or installing Compose as a container.
+2016, or Linux systems.
 
 > Install a different version
 >
 > The instructions below outline installation of the current stable release
```

Comment:
Wondering if we should replace / amend this to clearly mention these instructions are for v2, and link to github for v1 legacy install